dts/bindings/pinctrl/nxp,mcux-rt-pinctrl.yaml

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2021 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The node has the 'pinctrl' node label set in MCUX RT SoC's devicetree. These
+  node can be autogenerated using the MCUXpresso config tools combined with
+  the parse-rt-mex.py script in the NXP HAL.
```

Comment:
Would be good to extend this help, see for example ST/GD/Nordic compatible files.